src/ir/mod.rs

```diff
@@ -51,10 +51,6 @@ pub(crate) fn get_isa(jit: bool) -> Box<dyn TargetIsa + 'static> {
     flags_builder
         .enable("enable_verifier")
         .expect("enable_verifier should be a valid option");
-    // minimal optimizations
-    flags_builder
-        .set("opt_level", "speed")
```

Comment:
Should it be set to "none" instead, or is none a well defined default?

Reply:
'well defined' is a little unclear in the context of cranelift ... it's just as likely for `none` to be removed as for the default to be changed in the next release. As long as we have the test suite I'll notice so I think it's ok. See also https://github.com/bytecodealliance/wasmtime/issues/1634

Reply:
Cranelift releases are fun.